src/main/java/duke/command/Command.java

```diff
@@ -0,0 +1,31 @@
+package duke.command;
+
+import duke.exception.DukeException;
+import duke.gui.Ui;
+import duke.task.TaskList;
+import duke.util.Storage;
+
+/**
+ * Represents the command that is executed when the user inputs a command.
+ *
+ * @author njxue
+ * @version v0.1
+ */
+public abstract class Command {
+    /**
+     * Executes the corresponding command.
+     *
+     * @param tasks TaskList containing the list of tasks.
+     * @param ui Ui object which interacts with the user.
+     * @param storage Storage object which loads and saves tasks.
+     * @throws DukeException If an exception occurs.
+     */
+    public abstract String execute(TaskList tasks, Ui ui, Storage storage) throws DukeException;
```

Comment:
I see that you have forgotten to add `@return` Javadoc for this method. 
Was it something you accidentally neglected while refactoring for Level-10 JavaFX?

Reply:
Nice catch! No wonder you're an A+ student!